src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/train/fluid.py

```diff
@@ -277,7 +278,5 @@ def mix_in_stream(
 
         return FluidStream(
             existing_fluid=mixed_neqsim_fluid_stream,
-            temperature_kelvin=temperature_kelvin,
```

Comment:
so, this was a bug?

Reply:
or just unnecessary?

Reply:
Just unnecessary. When existing_fluid is defined, the pressure and temperature is taken from there. The bug was line 61 and 62. pressure_bara and temperature_kelvin was always defaulted to UnitConstants.STANDARD_PRESSURE_BARA and UnitConstants.STANDARD_TEMPERATURE_KELVIN, so the if statement would always evaluate to true...